src/libclipper/src/redis.cpp

```diff
@@ -438,6 +454,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
```

Comment:
We resolved to just have all the checks be in the Management Frontend and to move them to Redis in another PR